assets/openapi.yaml

```diff
@@ -911,6 +1178,103 @@ paths:
+    get:
+      security:
+        - BasicAuth: []
+      summary: Retrieves a list of orders for the account, filtered by the supplied query parameters.
+      tags:
+        - Trading
+      description: Retrieves a list of orders for the account, filtered by the supplied query parameters.
+      parameters:
+        - $ref: "#/components/parameters/AccountID"
+      requestBody:
+          required: false
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    enum:
+                      - open
+                      - closed
+                      - all
+                    example: all
+                  limit:
+                    type: integer
+                    example: 500
+                  after:
+                    type: string
+                    format: date-time
+                    example: "2021-03-16T18:38:01.942282Z"
+                  until:
+                    type: string
+                    format: date-time
+                    example: "2021-03-16T18:38:01.942282Z"
+                  direction:
+                    type: string
+                    enum:
+                      - asc
+                      - desc
+                    example: desc
+                  nested:
+                    type: boolean
+                    example: false
+                  symbols:
+                    type: string
+                    example: AAPL
+      responses:
+        "200":
+          description: Retrieves a list of orders for the account, filtered by the supplied query parameters.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/OrderObject"
+        "400":
+          $ref: "#/components/responses/Invalid"
+        "404":
+          $ref: "#/components/responses/NotFound"
+    post:
+      security:
+        - BasicAuth: []
+      summary: Create an order for an account.
+      tags:
+        - Trading
+      description: Create an order for an account.
+      parameters:
+        - $ref: "#/components/parameters/AccountID"
+      requestBody:
+          required: true
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/CreateOrder"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  id:
+                    type: string
+                    example: c8f1ef5d-edc0-4f23-9ee4-378f19cb92a4
+                  account_number:
+                    type: string
+                    example: 927584925
+                  status:
+                    $ref: "#/components/schemas/OrderObject"
```

Comment:
it does not give back an object with id,account_number,status, just the orderentity